js/content/common.js

```diff
@@ -1166,7 +1166,7 @@ let AugmentedSteam = (function() {
         HTML.afterBegin("#global_action_menu",
             `<div id="es_menu">
                 <span id="es_pulldown" class="pulldown global_action_link">Augmented Steam</span>
-                <div id="es_popup" class="popup_block_new">
+                <div id="es_popup" class="popup_block_new" style="display: none;">
```

Comment:
Styles should be kept in CSS files if possible

Reply:
But `ShowMenu()` will take control of this after clicking on the popup, is it still necessary?

Reply:
Yes, I'd say so.
Having styles split up over JS and CSS is not a good practice